okhttp/src/test/java/okhttp3/KotlinSourceCompatibilityTest.kt

```diff
@@ -943,31 +943,31 @@ class KotlinSourceCompatibilityTest {
     var recordedRequest: RecordedRequest = RecordedRequest(
         "", Headers.of(), listOf(), 0L, Buffer(), 0, Socket())
     recordedRequest = RecordedRequest("", Headers.of(), listOf(), 0L, Buffer(), 0, Socket())
-    var requestUrl: HttpUrl? = recordedRequest.requestUrl
-    var requestLine: String = recordedRequest.requestLine
-    var method: String? = recordedRequest.method
-    var path: String? = recordedRequest.path
-    var headers: Headers = recordedRequest.headers
+    var requestUrl: HttpUrl? = recordedRequest.getRequestUrl()
```

Comment:
This file shouldn't be changed. Only the "modern" one should be updated.

Reply:
ah, I thought `KotlinSourceCompatibilityTest` tests _deprecated_ functions and `KotlinSourceModernTest ` tests _idiomatic_ functions. But it sounds like I'm probably wrong?

Reply:
It tests how Kotlin compiles against OkHttp 3.x.

Reply:
You are both right, the file got cleaned up in https://github.com/square/okhttp/pull/5003#discussion_r280961308

@swankjesse ok'd it

Reply:
Yeah, the goal of this file is to have a record of all of the source-incompatible changes we made so we can either avoid them or flag them to users.